crates/client/analytics/src/lib.rs

```diff
@@ -42,7 +42,7 @@ impl Analytics {
 
         let tracing_subscriber = tracing_subscriber::registry()
             .with(tracing_subscriber::filter::LevelFilter::from_level(self.log_level))
-            .with(tracing_subscriber::fmt::layer().event_format(custom_formatter));
+            .with(tracing_subscriber::fmt::layer().with_writer(std::io::stderr).event_format(custom_formatter));
```

Comment:
What does this change do? Logging to stderr?

Reply:
yep
it's a fix for the pr that brought tracing in, the logs are now in stdout instead of stderr
its a subtle change but its kinda a breaking change when you think about it, i'm just reverting it
and that actually bothers me because the e2e tests inherit stderr but pipe stdout, and i couldnt see anything of the node logs when adding e2e tests

Reply:
But shouldn't logs print out to stdout?

Reply:
it makes more sense to have logs be printed in stderr right?

Reply:
ok sure. Plus as you say this is reverting behaviour back.

Reply:
in my opinion, stdout should be reserved for program output, while stderr is best suited for diagnostics, including logs